packages/hooks-react/src/useOffers.ts

```diff
@@ -34,7 +34,7 @@ const useOffers = () => {
   const switchSubscription = useMutation({
     mutationKey: ['switchSubscription'],
     mutationFn: checkoutController.switchSubscription,
-    onSuccess: () => accountController.reloadSubscriptions({ delay: 7500 }), // @todo: Is there a better way to wait?
+    onSuccess: () => accountController.reloadSubscriptions({ delay: 3000, retry: 10 }), // A subscription switch usually takes at least 3 secs
```

Comment:
Will it work properl if we already have an "activeSubscription"? So it looks like this condition `!activeSubscription && !!retry && retry > 0` won't work.

Reply:
Good point, probably not. We could fix it using the approach proposed by @dbudzins in #483, but that's too large of a refactor for this moment. I have noted this for another moment.

But until that, for now I see two options:
- We set the initial delay for this case back to 7500, and merge this PR as it is still a step forward
- We set this PR back into Draft for now

I'm fine with both options. :)

Reply:
I am good with the first option. Just wanted to make sure this is something you are aware of.